package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "0.7.0",
+  "version": "0.7.1",
```

Comment:
I follow an unusual pattern at the moment. I increase the patch version while things goes into testing. I increase the minor one when things go to release... 
This is a temporary step until "we" reach 1.0.0 and then "we" can start following the normal semver versioning. 
I might introduce semantic-release to this project to automate this even more.